src/main/java/com/sports/server/command/timeline/domain/Timeline.java

```diff
@@ -18,12 +18,14 @@ public abstract class Timeline extends BaseEntity<Timeline> {
 
     @ManyToOne(fetch = FetchType.LAZY)
     @JoinColumn(name = "game_id", nullable = false)
-    private Game game;
+    protected Game game;
 
     @ManyToOne(fetch = FetchType.LAZY)
     @JoinColumn(name = "recorded_quarter_id", nullable = false)
-    private Quarter recordedQuarter;
+    protected Quarter recordedQuarter;
 
     @Column(name = "recorded_at", nullable = false)
-    private Integer recordedAt;
+    protected Integer recordedAt;
+
+    abstract public String getType();
```

Comment:
```suggestion
    public abstract String getType();
```
제어자 순서를 컨벤션에 맞게 바꾸는게 좋을거 같아!

[Modifier Keyword order in Java](https://stackoverflow.com/questions/10299067/modifier-keyword-order-in-java)
[Java SE Classification](https://docs.oracle.com/javase/specs/jls/se7/html/jls-8.html#jls-8.3.1)